Networking/Networking/Mapper/SiteSettingMapper.swift

```diff
@@ -0,0 +1,43 @@
+import Foundation
+
+/// Mapper: SiteSetting
```

Comment:
I would say to add a description here, where it is explained that unlike `SiteSettingsMapper`, this mapper is used to parse a single setting.

Reply:
Updated in 2c7e5478c3351069e2b17f047f2d87c17f8b41a0.